perllib/FixMyStreet/App/Controller/My.pm

```diff
@@ -261,13 +261,36 @@ sub shortlist_multiple : Path('planned/change_multiple') {
     $c->forward('/auth/check_csrf_token');
 
     my @ids = $c->get_param_list('ids[]');
-
-    foreach my $id (@ids) {
-      $c->forward( '/report/load_problem_or_display_error', [ $id ] );
-      $c->user->add_to_planned_reports($c->stash->{problem});
+    my @bulk_reports = $c->get_param_list('bulk-assign-reports');
```

Comment:
I'm not sure why this uses the same controller method as the shortlist-all-pins functionality - they don't appear to share anything bar the CSRF check and fetching a list of ids (which aren't even used in the new assign code). It would be clearer if it was it owns method/form endpoint.